Agregado nuevo TDA "Casillero"

NLGS2907 opened the pull request:
Agregué, aunque incompleto, un nuevo TDA "Casillero", que serán a partir de ahora el dato del tablero. Lo ideal sería completar métodos que ayuden con el algoritmo de búsqueda.

NLGS2907 committed:
Actualizado el Readme.md

Actualizado para hacer saber que saúl dejó de estar en el equipo.

NLGS2907 committed:
Actualizado tablero.h, creado casillero.h

Creé el TDA "casillero", que es el dato que a partir de ahora va a guardar el tablero.

NLGS2907 committed:
Merge branch 'main' into FrancoL

guillehoff commented:
Hola

guillehoff reviewed (approved):
Funciono?

NLGS2907 commented:
asi parece, solo faltaria uno mas

nicogareca commented on casillero.h:
```diff
@@ -0,0 +1,55 @@
+template <class TipoCasillero>
+class Casillero {
+
+    private:
```
Creo que alguno de los atributos para casillero son: 
-char fichaJugador;
-EstadoCasillero estado; (con un enum)
-Coordenada coordenadaCasillero; (este me parece necesario, lo q si no se si esta bien si hay q hacer un tda coordenada)
Obviamente va a q haber agregar  metodos para manejar estos atributos

NLGS2907 replied:
si por `char fichaJugador` te referís a la letra que representa una ficha dentro del casillero, eso lo entendí como el contenido y ya está implementado en `tipoCasillero contenido`.
ahora mismo no se me ocurre qué estados tendría además de "vacío" o "ocupado" en los casilleros, por lo que no veo necesidad de un enum (lo vemos mejor después).
No llego a entender para qué querés una coordenada? (y lo que es más un TDA, juraría que no hace falta).

nicogareca commented on tablero.h:
```diff
@@ -28,7 +29,13 @@ class Tablero {
         especificadas en el constructor.
         */
-        Tablero(int altoInic, int anchoInic, int largoInic);
+        Tablero(int anchoInic, int altoInic, int largoInic);
```
Hay q poner las palabras completas de "altoInicial" y las otras por q gustavo desp se enoja.

NLGS2907 replied:
ok, ahi lo cambio

NLGS2907 committed:
Retocados algunos nombre de variables

En el TDA Tablero, el constructor y otras funciones tienen los nombres del tipo `altoInic` cambiado a `altoInicial`.
Cambié el directorio de donde el readme.md saca los GIFs para organizar mejor la carpeta donde están contenidas y prepararla para cuando hagamos el informe.

NLGS2907 commented:
ahi retoqué nombre de variables nico, fijate ahora
